solr/core/src/java/org/apache/solr/util/hll/HLL.java

```diff
@@ -44,10 +44,10 @@
  *
  * This data structure is interoperable with the implementations found at:
  * <ul>
- *   <li><a href="https://github.com/aggregateknowledge/postgresql-hll">postgresql-hll</a>, and</li>
+ *   <li><a href="https://github.com/citusdata/postgresql-hll/blob/master/README.md">postgresql-hll</a>, and</li>
  *   <li><a href="https://github.com/aggregateknowledge/js-hll">js-hll</a></li>
  * </ul>
- * when <a href="https://github.com/aggregateknowledge/postgresql-hll/blob/master/STORAGE.markdown">properly serialized</a>.
+ * when <a href="https://github.com/metdos/postgresql-hll/blob/master/STORAGE.markdown">properly serialized</a>.
```

Comment:
It feels a bit strange that we have a July-2015 fork of the hll library in our code base, and then link to live, changing documentation for two different forks of the original library. 

Better link to the citrusdata repo and a specific version back in time, like https://github.com/citusdata/postgresql-hll/tree/v2.9.0 which is the latest version that seems to have STORAGE.markdown. May not be 100% exact but close enough.